lambdas/sessionstart/src/main/java/uk/gov/di/ipv/core/session/IpvSessionStartHandler.java

```diff
@@ -159,6 +176,12 @@ private ClientSessionDetailsDto generateClientSessionDetails(
                 isDebugJourney);
     }
 
+    @Tracing
+    private ClientSessionDetailsDto generateErrorClientSessionDetails(
+            String redirectUri, String clientId) {
+        return new ClientSessionDetailsDto(null, clientId, redirectUri, null, null, false);
```

Comment:
I think we need to include the state value in this if one exists, so we can pass it back with the error and error_description params.

Reply:
Is that a thing to return the state in a Oauth error response? I couldn't see anything about it in the Oauth RFC - https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1

Reply:
As discussed, spotted that it does list state on the next page of the RFC. I'll update this PR to return a state param if it exists during an error response